package.json

```
@@ -1,6 +1,6 @@
 {
     "name": "@salesforce/sfdx-lwc-jest",
-    "version": "0.11.0",
+    "version": "0.12.0",
```

Keep the release version the same. Not required to update the version for bumping dependencies. Can be done before publishing.
```suggestion
    "version": "0.11.0",
```